corehq/apps/geospatial/forms.py

```diff
@@ -85,3 +86,105 @@ def __init__(self, *args, **kwargs):
                 )
             )
         )
+
+
+class ConfigureCaseGroupingForm(forms.ModelForm):
+
+    class Meta:
+        model = GeoConfig
+        fields = [
+            "selected_grouping_method",
+            "max_cases_per_group",
+            "min_cases_per_group",
+            "target_group_count",
+            "selected_disbursement_algorithm",
+        ]
+
+    selected_grouping_method = forms.ChoiceField(
+        label=_("Grouping method"),
+        help_text=_("Determines which parameter to use for grouping cases"),
+        required=False,
+        choices=GeoConfig.VALID_GROUPING_METHODS,
+    )
+    max_cases_per_group = forms.IntegerField(
+        label=_("Maximum group size"),
+        help_text=_("The minimum number of cases that can be in a group"),
+        required=False,
+        min_value=1,
+    )
+    min_cases_per_group = forms.IntegerField(
+        label=("Minimum group size"),
+        help_text=_("The maximum number of cases that can be in a group"),
+        required=False,
+        min_value=1,
+    )
+    target_group_count = forms.IntegerField(
+        label=_("Target group count"),
+        help_text=_("The desired number of groups. Cases will be divided equally to create this many groups"),
+        required=False,
+        min_value=1,
+    )
+    selected_disbursement_algorithm = forms.ChoiceField(
+        label=_("Disbursement algorithm"),
+        # TODO: Uncomment and add documentation link when confluence page for algorithms has been created
```

Comment:
Is [this](https://confluence.dimagi.com/pages/viewpage.action?pageId=164694245#:~:text=them%20as%20necessary.-,Radial%20Cost,-This%20is%20implemented) perhaps the documentation you're referring to here?

Reply:
Ah! I didn't know the documentation was already done for this. I do see it's not under CommCare public however, so I'm hesitant towards linking to it on HQ until it is. But I'll update the code comment to reflect this.